docs/development/heat_dissipation.md

```diff
@@ -1,11 +1,92 @@
-# Heat dissipation
-
-::: warning TODO
-- Description of losses in power electronics
-    - Switching losses
-    - Conduction losses
-- Methods for cooling
-    - Passive using PCB
-    - Passive via heat sink
-    - Active (forced convection)
-:::
+# Losses and Heat
+
+In every real electrical system, flowing currents are generating heat inside the components. While most low energy devices do not generate siginificant amount of thermal waste, power electronics need to dissipate large amount of thermal energy.
+
+There are many different components on a board that contribute to the overall power loss, like resistors, traces, fuses, coil resistance and other ohmic resistances. In circuits with frequent switching, the impedance of capacitators and inductances can play a role as well, but won't be treated here.
+
+The main factor for heat generation in power electronics are the semi-conductors, mainly **MOSFETS** used for switching. In the following, switching and conduction losses for **MOSFETs** are explained and different ways to dissipate the heat are given.
+
+## Conduction loss
+
+Conduction loss appears in the steady-state phase when the system is not changing anymore after being switched on. The power can be calculated using
+
+$$ P_{cond} = r_{ds} \cdot I^2 $$
```

Comment:
- Correct symbol for heat is Q, not P (even though the unit is the same).
- R in capital letters. On-state resistance typically written as below.
(please fix also in other locations)
```suggestion
$$ Q_{cond} = R_{DS(on)} \cdot I^2 $$
```